Client/Frontend/Widgets/SearchInputView.swift

```diff
@@ -22,7 +22,7 @@ private struct SearchInputViewUX {
     func searchInputViewFinishedEditing(_ searchView: SearchInputView)
 }
 
-class SearchInputView: UIView {
+class SearchInputView: UIView, Themeable {
```

Comment:
Although this will fix the issue with the colors, this search bar will still appear very different from the one in Data Management. That's ok though for now I think. We should maybe file a follow-up to make this consistent.

Reply:
Yup, I put the follow up bugs in the description: https://github.com/mozilla-mobile/firefox-ios/pull/4458#issue-243024490